src/main/java/com/teamcautionrobotics/robot/Robot.java

```diff
@@ -139,13 +142,20 @@ public void robotPeriodic() {
         } else {
             cargo.intake(CargoMoverSetting.STOP);
         }
-
+        
         if (deployedFunnelRoller != driverRight.getTrigger() && driverRight.getTrigger()) {
             cargo.toggleFunnelRoller();
         }
         deployedFunnelRoller = driverRight.getTrigger();
-
+        
         cargo.deployExitFlap(driverLeft.getTrigger());
+
+        if (jackButtonPressed != manipulator.getButton(Button.X) && manipulator.getButton(Button.X)) {
+            habJack.switchState();
+        }
+        jackButtonPressed = manipulator.getButton(Button.X);
+
+        aimingLightsButtonPressed = driverLeft.getRawButton(2);
```

Comment:
Is there a reason for moving this?

Reply:
To keep consistent the order of instances we do stuff with in the class itself (declaration), robotInit (initialization), and robotPeriodic (doing stuff).